galasa-parent/dev.galasa.framework.api.cps/openapi.yaml

```diff
@@ -0,0 +1,57 @@
+openapi: 3.0.0
+info:
+  title: Galasa CPS Requests
+  version: 0.0.1
+
+servers:
+  - url: 'http://{server}:{port}'
+    variables:
+      server:
+        default: localhost
+      port:
+        default: "8080"
+
+paths:
+  /cps/{namespace}:
```

Comment:
will need a
GET /cps/namespace/ which will get all the namespaces
GET /cps/namespace/{namespace}/   which will return all properties of the namespace
GET /cps/namespace/{namespace}/prefix/{prefix}/suffix/{suffix}?infix=xxx?infix=xxx?infix=xxx

return an appropriate json object,      if returning a value,  return the property name as well.

Reply:
PUT /cps/namespace/{namespace}/property/{property}